### ISSUE TITLE
11.50 support?

### ISSUE THREAD

**Issue by Ishikudeska** [GoldHEN/GoldHEN#173]
It's the newest version, but I understand if y'all want to wait until something like 12.00 comes.

Sony needs to fuck off and stop updating PS4 firmware (or give bloodborne PC)!

**Comment by Ishikudeska** [comment]
I am willing to help with this, but I need to learn HOW to do it.

**Comment by Masamune3210** [comment]
Fun Fact, they can't stop making firmwares for the PS4 until either

A. they stop caring about people's ability to use it as a Blu-Ray player -  they have to keep routinely updating the Blu-Ray decryption keys if they want it to be able to play new Blu-Ray discs

B. it stops being able to connect to PSN - They have to keep various things up to date with the service, such as certs and security expectations.

**Comment by Masamune3210** [comment]
and yes, this is why the ps3 firmware still gets updates, those pesky blu-ray keys have to be kept in sync

**Comment by Ishikudeska** [comment]
Technically bcz the PS3 shop is shut down, and they don't make the PS3 (or Ps3 discs). They have no ownership over the PS3 as it's not "commercially available from an official source"
The PS4 doesn't fall under this until they shut down the store, or stop producing them.

**Comment by Masamune3210** [comment]
Just because they arent selling them anymore....doesnt mean they arent fully aware people are still using them.

**Comment by Ishikudeska** [comment]
Right, They're aware based on how many people download the new firmware updates, or have their console connected to the internet.

**Comment by Ishikudeska** [comment]
If not already jailbroken

**Comment by Shuppama** [comment]
For now id say to forget about ur ps4, play on pc or xbox, when next goldhen version releases grab ur ps4 again
